models/DataObject/ClassDefinition/Data/Datetime.php

```diff
@@ -309,24 +311,23 @@ public function getFromWebserviceImport($value, $object = null, $params = [], $i
         $timestamp = strtotime($value);
         if (empty($value)) {
             return null;
-        } elseif ($timestamp !== false) {
+        }
+        if ($timestamp !== false) {
             return $this->getDateFromTimestamp($timestamp);
-        } else {
-            throw new \Exception('cannot get values from web service import - invalid data');
         }
+        throw new \Exception('cannot get values from web service import - invalid data');
     }
 
     /**
-     * @return Date
+     * @return int
      */
     public function getDefaultValue()
     {
         if ($this->defaultValue !== null) {
```

Comment:
existing code looks wrong somehow

Reply:
can be simply `return $this->defaultValue`, or?

Reply:
my interpretation is that

public $defaultValue

should be int | null.

Setter do the conversion (in case of string for example).

Getter should return $this->defaultValue as it is.

Reply:
I changed it, the setter seems to be ok.